bin/cheribsdtest/Makefile.cheribsdtest

```diff
@@ -30,6 +30,7 @@ SRCS+=	cheribsdtest_bounds_globals.c					\
 .if ${MACHINE_ABI:Mpurecap}
 SRCS+=	cheribsdtest_bounds_subobject.c					\
 	cheribsdtest_bounds_varargs.c					\
+	cheribsdtest_malloc.c						\
```

Comment:
This won't work without MRS?

Reply:
There's a stub malloc_is_revoking() so they will skip with non-mrs allocators. While we could disable it to avoid skipped tests, I hope to move away from mrs at some point in the not to distant future.

Reply:
Ok, that's fine then